.dockerignore

```diff
@@ -0,0 +1,10 @@
+LICENSE
```

Comment:
Do we need the .dockerignore file to be committed?

Reply:
We don't need them at this stage, I'll take both Dockerfile and .dockerignore out for now 😉